packages/backend/src/income/__tests__/handler/createIncomeHandler.test.ts

```diff
@@ -0,0 +1,63 @@
+import { createIncomeService } from '../../service/createIncomeService'
+import { validateCreateIncomeInput } from '../../validation/'
+import type { Context } from 'koa'
+import { STATUS_CREATED_201 } from '../../../models/statusCodes'
+import { createIncomeHandler } from '../../handler/createIncomeHandler'
+
+jest.mock('../../service/createIncomeService')
+jest.mock('../../validation')
+
+const mockService = createIncomeService as jest.Mock
+const mockValidation = validateCreateIncomeInput as jest.Mock
+
+describe('createIncomeHandler', () => {
+  const fakeCtx = {
+    request: {
+      body: {},
+    },
+    status: undefined,
+    body: undefined,
+  } as unknown as Context
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+  })
+
+  describe('on success', () => {
+    it('should return 201 and expense', async () => {
+      // Arrange
+      mockService.mockResolvedValue({ id: 1, name: 'Lunch' })
+
+      // Act
+      await createIncomeHandler(fakeCtx)
+
+      // Assert
+      expect(fakeCtx.status).toBe(STATUS_CREATED_201)
+      expect(fakeCtx.body).toEqual({ createdIncome: { id: 1, name: 'Lunch' } })
```

Comment:
nit: make `{id:1, name: Lunch}` into a variable and reuse it between the mockResolved and the assertion